test/integration/pipeline/tx_pipeline_test.cpp

```diff
@@ -0,0 +1,62 @@
+/**
+ * Copyright Soramitsu Co., Ltd. 2018 All Rights Reserved.
+ * http://soramitsu.co.jp
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <gmock/gmock.h>
+#include <gtest/gtest.h>
+#include "crypto/keys_manager_impl.hpp"
+#include "framework/integration_framework/commands.hpp"
+#include "framework/integration_framework/integration_test_framework.hpp"
+#include "framework/integration_framework/utils.hpp"
+
+using integration_framework::IntegrationTestFramework;
+using iroha::protocol::TxStatus;
```

Comment:
There is no need to depend on protocol buffers in integration test. Model object can be passed to validation function in order to satisfy this.